node/package.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Any specific reason to use npm instead of yarn?

Reply:
removed lock file.